### PR TITLE
 [Libretro] Fix some default config values

### LINKED ISSUE #12144: No audio with latest versions of the core in Retroarch
### What happens?

Games load with no audio on the most recent version of the core (~6/24 or newer). Also reported here: https://forums.libretro.com/t/no-audio-for-ppsspp-core/23022/4

### What *should* happen?

Games should have audio. Most recent version of the core with audio verified was 6/22/19.

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.

Windows 10 x64, I5-9400f, GTX 1660ti. Tested with the most recent nightly retroarch.

### PR DESCRIPTION
**Pull request by m4xw**
-iAltSpeedVolume
-iFirmwareVersion
-iPSPModel
Fixes #12144

### CODE REVIEW

**Comment by unknownbrackets** [comment]
This should *really* use the defaults in Config.cpp.  Replicating them just means things will break again.

-[Unknown]

**Comment by m4xw** [comment]
Agree, but this is just following existing code for now.
It will need some more extensive refactor in general.
I think this is one of the bigger reasons why weird libretro quirks happen to begin with.
Will probably start rewriting some of the libretro specific code next week, depending on how far I come with my other projects (already wanted to finish the switch port 2 weeks ago..., oh well)

**Comment by unknownbrackets** [comment]
Other settings likely to cause problems:

 * sMacAddress (problems with netplay)
 * iNumWorkerThreads (will make various things slower on dual core+ devices)
 * bIgnoreBadMemAccess (probably makes hangs/crashes more common)
 * bSeparateSASThread (will make sound effects slightly slower)

Just from a quick glance at some, there are probably a bunch more.  Many of these have code that determines an appropriate default per device.

-[Unknown]

File: libretro/libretro.cpp [hunk 1]
```diff
@@ -321,10 +321,14 @@ void retro_init(void) {
 	g_Config.bFrameSkipUnthrottle = false;
 	g_Config.bMemStickInserted = PSP_MEMORYSTICK_STATE_INSERTED;
```
Comment (unknownbrackets) [inline libretro/libretro.cpp]: This is a boolean, should just be true.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

File: libretro/libretro.cpp (hunk 1 above)
Comment (m4xw) [reply libretro/libretro.cpp]: Will change it next upstream PR

**Pull request merged**